torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1463,8 +1500,7 @@ def sample_inputs_masked_select(op_info, device, dtype, requires_grad):
                    aliases=('arcsin', ),
                    ref=np.arcsin,
                    domain=(-1, 1),
-                   supports_sparse=True,
-                   decorators=(precisionOverride({torch.bfloat16: 1e-2}),),
+                   decorators=(precisionOverride({torch.bfloat16: 1e-2, torch.float16: 5e-4}),),
```

Comment:
What is the half precision change needed?

Reply:
It is a failure on cuda:
```
test_function_consistency_asin_cuda_float16

AssertionError: False is not true : Tensors failed to compare as equal!With rtol=0.001 and atol=1e-05, found 1 element(s) (out of 29925) whose difference(s) exceeded the margin of error (including 0 nan comparisons). The greatest difference was 0.00048828125 (-0.216064453125 vs. -0.216552734375), which occurred at index (53, 37).  ```

Reply:
It's fine as is, I was just curious.